ui/core/proto_utils/action_id.ts

```diff
@@ -373,6 +373,13 @@ export class ActionId {
 					name += ` (self)`;
 				}
 				break;
+			case 'Stormlash':
+				if (tag === -1) {
+					name += ' (raid)';
+				} else {
+					name += ` (self)`;
+				}
+				break;
```

Comment:
Group this with the Shattering Throw case so you don't have to copy-paste the logic.